go/pools/resource_pool.go

```diff
@@ -238,21 +206,13 @@ func (rp *ResourcePool) reopen() {
 // it will wait till the next resource becomes available or a timeout.
 // A timeout of 0 is an indefinite wait.
 func (rp *ResourcePool) Get(ctx context.Context) (resource Resource, err error) {
-	span, ctx := trace.NewSpan(ctx, "ResourcePool.Get")
```

Comment:
this seems unrelated. Why remove the tracing here?

Reply:
The trace is already happening at one higher up level in connpool.Pool
Seems like a unnecessary trace.

```
	span.Annotate("capacity", p.Capacity())
	span.Annotate("in_use", p.InUse())
	span.Annotate("available", p.Available())
	span.Annotate("active", p.Active())

	if cp.timeout != 0 {
		var cancel context.CancelFunc
		ctx, cancel = context.WithTimeout(ctx, cp.timeout)
		defer cancel()
	}
	r, err := p.Get(ctx, nil)
```